modules/gitops-configuring-gitops-managed-resources-to-use-mounted-secrets.adoc

```diff
@@ -0,0 +1,117 @@
+// Module is included in the following assemblies:
+//
+// * securing_openshift_gitops/managing-secrets-securely-using-sscsid-with-gitops.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-configuring-gitops-managed-resources-to-use-mounted-secrets_{context}"]
+= Configuring {gitops-shortname} managed resources to use mounted secrets
+
+You must configure the {gitops-shortname} managed resources by adding volume mounts configuration to a deployment and configure the container pod to use the mounted secret. For example, consider that you want to add volume mounts configuration to the deployment of `app-taxi` application and the `100-deployment.yaml` file is in the `/environments/dev/apps/app-taxi/services/taxi/base/config/` directory.
+
+.Prerequisites
+
+* You have the AWS Secrets Manager resources stored in your {gitops-shortname} repository.
+* You have the Secrets Store Container Storage Interface (SSCSI) driver configured to mount secrets from AWS Secrets Manager.
+
+.Procedure
+
+. Add volume mounts configuration in the target resource:
+
+.. Create a `100-deployment.yaml` file to use the secret provider class resources:
++
+.Example `100-deployment.yaml` file
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  labels:
+    app.kubernetes.io/name: taxi
+    app.kubernetes.io/part-of: app-taxi
+    name: taxi                                # <1>
+    namespace: dev                            # <2>
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+    app.kubernetes.io/name: taxi
+    app.kubernetes.io/part-of: app-taxi
+  strategy: {}
+  template:
+    metadata:
+    creationTimestamp: null
+    labels:
+      app.kubernetes.io/name: taxi
+      app.kubernetes.io/part-of: app-taxi
+    spec:
+      containers:
+        - image: nginxinc/nginx-unprivileged:latest
+          imagePullPolicy: Always
+          name: taxi
+          ports:
+            - containerPort: 8080
+          volumeMounts:
+            - name: secrets-store-inline
+              mountPath: "/mnt/secrets-store"
+              readOnly: true
+          resources: {}
+    serviceAccountName: default
+    volumes:
+      - name: secrets-store-inline
+        csi:
+          driver: secrets-store.csi.k8s.io
+          readOnly: true
+          volumeAttributes:
+          secretProviderClass: "my-aws-provider" # <3>
+    status: {}
+----
+<1> Name of the deployment.
+<2> Namespace for the deployment. This must be the same namespace as the secret provider class.
+<3> Name of the secret provider class.
+
+.. Push the `100-deployment.yaml` file to your GitOps repository.
+
+. In the Argo CD UI, click *REFRESH* on the target application page to apply the updated deployment manifest.
+
+. Verify that all the resources are successfully synchronized on the target application page, for example, on the `dev-app-taxi` application page.
```

Comment:
I feel this `dev-app-taxi` example is confusing for someone who's not familiar with the KAM generated repo. Can you avoid using it in this entire documentation?